pages/support.tsx

```diff
@@ -0,0 +1,241 @@
+        <meta
+          property="description"
+          key="description"
+          content="Our mission is to empower creators and collectors with a suite of tools to create, market, and sell NFTs."
+        />
+        <body />
+      </Head>
+
+      <div className='relative'>
+        <div className='absolute z-[-100] w-full h-full top-0 left-[-10%] lg:w-[70%] lg:left-[15%] bg-no-repeat bg-contain bg-[url(/images/page-support/blob-01.svg)] pointer-events-none' />
+        <Section>
+          <Content className="text-center lg:my-12">
+            <div className='mx-auto max-w-5xl'>
+              <Heading>Solanas top community <br className='hidden lg:block' /> supported NFT platform</Heading>
+              <Paragraph>
+                Web3 is an exciting place but sometimes we need help. We rely on collaboration from members from the community to provide assistance. Get involved by contributing to our wiki page or in our Discord.
+              </Paragraph>
+              <a target='_blank' rel='noreferrer' href='https://docs.holaplex.com/'><Button>Read The Docs</Button></a>
+            </div>
+          </Content>
+        </Section>
+      </div>
+
+      <div className="container mx-auto px-6 md:px-12 lg:my-12">
+        <Section>
+          <div className="w-full lg:w-1/2">
+            <Content>
+              <SubHeading>Need Help?</SubHeading>
+              <Heading>Get support from the community</Heading>
+            </Content>
+          </div>
+          <Content>
+            <div className='lg:flex lg:gap-4 w-full'>
+              <div>
+                <Paragraph>
+                  <b>Information at your fingertips</b><br />
+                  Our documentation is expanding by the day thanks to contributions from our community.<br />
+                  <a target='_blank' rel='noreferrer' href="https://wiki.holaplex.com/" className='text-[#F7CC34]'>Find your answer</a>
+                </Paragraph>
+              </div>
+              <div>
+                <Paragraph>
+                  <span className="font-bold">Get help from the community</span><br />
+                  Couldn’t find your answer from the documentation? Ask a member from the community.<br />
+                  <a target='_blank' rel='noreferrer' href="https://discord.com/invite/holaplex" className='text-[#F7CC34]'>Join the Discord</a>
+                </Paragraph>
+              </div>
+            </div>
+          </Content>
+        </Section>
+
+        <Section className='lg:my-12'>
+          <Content className="text-center">
+            <SubHeading>What product do you need help with?</SubHeading>
+            <Heading>Support for all of the <br className='hidden lg:block' /> Holaplex NFT products</Heading>
+          </Content>
+
+          <Content>
+            <SupportRow>
+              <SupportImage>
+                <SupportBlob><Image src='/images/page-support/blob-02.svg' layout='fill' alt='' /></SupportBlob>
```

Comment:
please give a useful `alt` for images

Reply:
All images on this page are decorative or entirely redundant text-wise with the title/description next to them, an empty alt tag indicates that behavior is intentional to search engines and screen readers.